Card.cs

```diff
@@ -0,0 +1,77 @@
+﻿using EnumColors;
+using System;
+
+namespace Task1_GUI
+{
+    internal class Card : IComparable
+    {
+        public int Nominal { get; set; }
+        public Colors.SevenColors Color { get; set; }
+        public static bool IsCardCorrect { get; set; }
+        public Card() { }
+        public Card(string combination)
+        {
+            IsCardCorrect = CheckEnteredCombination(combination);
+        }
+        public bool Equals(Card card)
+        {
+            return (Nominal == card.Nominal && Color == card.Color);
+        }
+        public override string ToString()
+        {
+            return (Nominal + " " + Color);
+        }
+        private bool CheckEnteredCombination(string combination)
+        {
+            bool isDataCorrect = true;
+            const int nominalPosition = 0, colorPosition = 2;
+            bool isCardNominalCorrect = int.TryParse(Convert.ToString(combination[nominalPosition]), out int enteredNominal);
+            if (!isCardNominalCorrect)
+            {
+                Service.ErrorMessage = "\nInvalid value of card nominal!";
```

Comment:
It's very very bad practice to have dependencies like this. Your `Card` class shouldn't know anything about `Service`.  Service should use Card and not the other way around. Use Exceptions here